docs/sample_config.yaml

```diff
@@ -663,6 +663,26 @@ uploads_path: "DATADIR/uploads"
 #
 #max_upload_size: 10M
 
+# The largest allowed size for a user avatar. Only enforced when
+# enable_media_repo is True. If not defined, no restriction will be
+# imposed.
+#
+# Note that this only applies to when an avatar is changed globally.
+# Per-room avatar changes are not affected. See allow_per_room_profiles
+# for disabling that functionality.
```

Comment:
Is this OK for dinsic?

Reply:
They currently have per-room avatar functionality disabled, thus all avatar changes must be made globally, which the new option will gatekeep.